upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -1399,15 +1398,21 @@ func (n *nodeUpConfigBuilder) BuildConfig(ig *kops.InstanceGroup, apiserverAddit
 	return config, bootConfig, nil
 }
 
-func getTasksCertificate(caTasks map[string]*fitasks.Keypair, name string, config *nodeup.Config, includeKeypairID bool) error {
-	cas, err := fi.ResourceAsString(caTasks[name].Certificates())
+func loadCertificate(keysets map[string]*fi.Keyset, name string, config *nodeup.Config, includeKeypairID bool) error {
```

Comment:
```suggestion
func loadCertificates(keysets map[string]*fi.Keyset, name string, config *nodeup.Config, includeKeypairID bool) error {
```

Since there can be many, especially during key rotation.